docs/user-feedback/feedback/strings.md

```diff
@@ -0,0 +1,27 @@
+# Feedback: Strings
```

Comment:
I wonder if perhaps you should omit this so as not to prime people by having read your view.  I think the feedback gathered can carry even more weight if you remain neutral as a facilitator of the process.

Reply:
Yeah makes sense.